lib/commands/general.js

```diff
@@ -13,6 +15,27 @@ commands.active = async function () {
   }
 };
 
+commands.getDeviceDateAndTime = async function(){
+  let spawnEnv = _.clone(process.env);
+  log.info('Attempting to capture iOS device date and time');
+  try {
+      let idevicedate = await fs.which('idevicedate');
+      log.info(`Found idevicedate: '${idevicedate}'`);
+      let proc = new SubProcess('idevicedate', ['-u', this.udid], {env: spawnEnv});
+      proc.on('output', stdout => {
+        if (stdout) {
+        idevicedate = stdout.trim();
+        }
+      });
+      await proc.start();
+      await proc.stop();
```

Comment:
I think for this case we can use teen_process.exec instead of SubProcess. `exec` is just like SubProcess but instead of manually starting and stopping, you can just await it once.